src/Turbo/README.md

```diff
@@ -434,21 +439,23 @@ class Book { /* ... */ }
 ### Broadcast Options
 
 The `Broadcast` attribute comes with a set of handy options:
+// These are tightly coupled to Mercure currently: the Broadcast attribute is variadic. Can't we make this more generic and discoverable, with actual named arguments, at least for some of them?
 
--   `template` (`string`): Twig template to render (see above)
--   `topics` (`string[]`): Mercure topics to use, defaults to an array containing the Fully Qualified Class Name
--   `private` (`bool`): marks Mercure updates as private
--   `id` (`string`): `id` field of the SSE
--   `type` (`string`): `type` field of the SSE
--   `retry` (`int`): `retry` field of the SSE
+ - `template` (`string`): Twig template to render (see above)
+ - `topics` (`string[]`): Mercure topics to use, defaults to an array containing the Fully Qualified Class Name
+ - `private` (`bool`): marks Mercure updates as private
+ - `id` (`string`): `id` field of the SSE
+ - `type` (`string`): `type` field of the SSE
+ - `retry` (`int`): `retry` field of the SSE // what's the use case for these 3 SSE elements? Can't we remove them to keep this generic?
```

Comment:
It's useful for fine-tuning (`retry`)  and for advanced use cases such as Event Sourcing (`id` and `type`). The main benefit of the current implementation is that you can leverage all advanced features of the transport you use (regardless of if it's Mercure, Pusher, or another thing). I'm in favor of keeping this.